core/src/sql/v2/mod.rs

```diff
@@ -79,6 +79,7 @@ pub use self::base::Base;
 pub use self::block::Block;
 pub use self::bytes::Bytes;
 pub use self::cast::Cast;
+pub use self::change_feed_include::ChangeFeedInclude;
```

Comment:
```suggestion
pub use self::changefeed_include::ChangeFeedInclude;
```

Nit but for consistency!

Reply:
Yes, defo thanks! Will update all references